File: MultiRubroProducts/MultiRubroProducts/DTOS/Responses/ProviderResponse.cs
namespace MultiRubroProducts.DTOS.Responses
{
    public class ProviderResponse
    {
        public Guid ProviderId { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Address { get; set; } = string.Empty;
        public string Cellphone { get; set; } = string.Empty;
    }
}

File: MultiRubroProyectProducts/src/main/java/com/example/multirubroproyectproducts/responses/Provider/ProviderResponse.java
package com.example.multirubroproyectproducts.responses.Provider;

import com.example.multirubroproyectproducts.models.Product;
import com.example.multirubroproyectproducts.responses.Product.SimpleProductResponse;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;

import java.util.List;
import java.util.UUID;

@Data
@Getter
@Setter
public class ProviderResponse {

    UUID id;
    String name;
    String address;
    String cellphone;
    List<SimpleProductResponse> products;
}
